src/components/Person/Person.jsx

```diff
@@ -1 +1,32 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = (props) => {
+  const {
+    person,
+  } = props;
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      {age
+        ? (<p className="Person__age">{`I am ${age}`}</p>)
+        : null
+      }
```

Comment:
Good, but it shouldn't be rendered if there as no age. So try to use {age && (<...